build/gen_stub.php

```diff
@@ -2382,8 +2385,11 @@ public function initializeZval(string $zvalName): string
             if ($cExpr === '""') {
                 $code .= "\tZVAL_EMPTY_STRING(&$zvalName);\n";
             } else {
-                $code .= "\tzend_string *{$zvalName}_str = zend_string_init($cExpr, strlen($cExpr), 1);\n";
-                $code .= "\tZVAL_STR(&$zvalName, {$zvalName}_str);\n";
+                if ($forStringDef === '') {
+                    $forStringDef = "{$zvalName}_str";
+                }
+                $code .= "\tzend_string *$forStringDef = zend_string_init($cExpr, strlen($cExpr), 1);\n";
+                $code .= "\tZVAL_STR(&$zvalName, $forStringDef);\n";
```

Comment:
I suppose this can even be ZVAL_NEW_STR as they're not interned.

Reply:
Similarly, all strings that are coming from KNOWN_STRINGS can be set via ZVAL_INTERNED_STR